rotkehlchen/exchanges/exchange.py

```diff
@@ -507,3 +509,19 @@ def query_history_with_callbacks(
             )
         except RemoteError as e:
             fail_callback(str(e))
+
+    def send_unknown_asset_message(
+            self,
+            msg_aggregator: 'MessagesAggregator',
```

Comment:
Why pass it as argument? Interesting I see that it's not part of the `exchange.py` common class. Why is that? Have you found any exchange it's not passed? Would it make sense to move it to the common class? That would simplify things, no?

You should probably do this in a separate commit as this is gonna be a bit bigger and more easily separable change.